cmd/cosign/cli/options/certificate.go

```diff
@@ -42,13 +42,13 @@ func (o *CertVerifyOptions) AddFlags(cmd *cobra.Command) {
 	_ = cmd.Flags().SetAnnotation("certificate", cobra.BashCompFilenameExt, []string{"cert"})
 
 	cmd.Flags().StringVar(&o.CertEmail, "certificate-email", "",
```

Comment:
I'd just remove this flag actually. This is going to be going into 2.0, so we can break things.

Reply:
+1 -- I was thinking in the context of sigstore-python where we'll remove a whole bunch of things right before 1.0, but removal right now makes much more sense in the context of cosign 2.0.

Reply:
Done.